### PR TITLE
Support amzn image

### PR DESCRIPTION
**Pull request by jumping**
On the amazon images could not install by `python setup.py install`, the reason is the platform package could not detect the distribution from the platform.dist() which return 

```
>>> platform.dist()
('', '', '')
```

### CODE REVIEW

**Comment by bmhatfield** [comment]
Looks like setup.py needs some PEP8 cleanup.

Also, can you add the logrotate config in a separate PR from this one? These contexts seem to be unrelated.

File: setup.py [hunk 1]
```diff
@@ -59,6 +63,8 @@ def running_under_virtualenv():
                            glob('conf/collectors/*')))
         data_files.append(('/etc/diamond/handlers',
                            glob('conf/handlers/*')))
+        data_files.append(('/etc/logrotate.d',
+	                   ['debian/logrotate/diamond']))
```
Comment (jaingaurav) [inline setup.py]: I think you need to add another space here to fix the PEP8 issues.

**Comment by jaingaurav** [comment]
@bmhatfield Is the logrotate rule needed? I thought we manage log rotation ourselves ing the diamond config?

**Comment by bmhatfield** [comment]
@jaingaurav it may not be, which is one reason why I suggested separating the concern into a separate PR

**Comment by jumping** [comment]
@bmhatfield @jaingaurav Thank you.

**Comment by jumping** [comment]
I created another pull request https://github.com/python-diamond/Diamond/pull/360 for support amz, but for the logrotate function decide to not pull now.

**Comment by shortdudey123** [comment]
Closing since https://github.com/python-diamond/Diamond/pull/360 was merged and log rotate.d config is not needed

create a new PR if this is needed in the future

**Pull request closed without merging**